test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/IVsServicesTestCase.cs

```diff
@@ -14,6 +14,14 @@ namespace NuGet.Tests.Apex
 {
     public class IVsServicesTestCase : SharedVisualStudioHostTestClass, IClassFixture<VisualStudioHostFixtureFactory>
     {
+        private const string TestPackageName = "Contoso.A";
+        private const string TestPackageVersionV1 = "1.0.0";
+        private const string TestPackageVersionV2 = "2.0.0";
+        private const string PrimarySourceName = "source";
+        private const string SecondarySourceName = "SecondarySource";
+
+        private readonly SimpleTestPathContext _pathContext = new SimpleTestPathContext();
```

Comment:
Wouldn't this share the same pathContext across tests?
How does this get disposed?

Reply:
xunit news up an instance of this class for each test and then xunit calls `Dispose()` for the instance since it implements `IDisposable` which calls `_pathContext.Dispose()`

https://xunit.net/docs/shared-context